mobify/platform-scaffold
# [WEBDATA-127] Edit Cart functionality on SFCC Connector

# Edit Cart functionality on SFCC Connector

This PR adds the functionality to edit / update a cart item on the Salesforce Commerce Cloud Connector


 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-127

## Changes
- Adds new command to update cart items in both Merlin's & SFCC connectors
- Changes the configureURL on `Edit Cart` pages to have `item_id` in the URL instead of `basket_id`, as `basket_id` is available through session storage.

## Todos
- [ ] Ensure analytics changes make sense for new Merlin's command (maybe a new analytics action for updating cart)

## How to test-drive this PR
- Using both connectors:
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to cart, click Edit Cart, update the item, and ensure it works as intended
